### PR TITLE
#160280386 Social login component

### PR DESCRIPTION
**Pull request by anneKay**
### What does this PR do?

- Create social login component

### Description of Task to be completed?

    ft(social auth component): Social auth component

    - Write unit tests
    - create social login component
    - add links for signup
    - create styles for component

### How should this be manually tested?
- `git clone https://github.com/andela/fargo-ah-client.git Authors-Haven-Client`
- `npm install`
- `npm run start` the start script
- Go to '/login' path 

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?

- [#160280386](https://www.pivotaltracker.com/story/show/160280386)

### Screenshots (if appropriate)

image

### Questions:

N/A

### CODE REVIEW

File: src/styles/font.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$description-font: "Abel", sans-serif;
+$primary-font: "Lato", sans-serif;
+$title-font: "Raleway", sans-serif;
```
Comment (OKiMaureen) [inline src/styles/font.scss:3]: please add a new line below

File: src/components/LoginCard.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, Card, Icon } from 'semantic-ui-react';
+import { NavLink } from 'react-router-dom';
+import '../styles/loginCard.scss';
```
Comment (veeqtor) [inline src/components/LoginCard.js]: I thought we agreed on importing `scss` styles only in the `index.scss`, Please do well to import only in the index file, cheers.

File: src/styles/loginCard.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
+@import "./colours";
+@import "./font";
```
Comment (veeqtor) [inline src/styles/loginCard.scss]: And also look into this too.

File: src/components/LoginCard.js (hunk 2 above)
Comment (anneKay) [reply src/components/LoginCard.js]: This has been fixed now. Thanks

File: .stylelintrc [hunk 4]
```diff
@@ -6,3 +6,4 @@
     "indentation": 2,
     "unit-whitelist": ["px", "em", "%", "rem", "ms", "s", "vh"],
   }
+}
```
Comment (veeqtor) [inline .stylelintrc]: New Line Please.

**Review by veeqtor — APPROVED** [review]

**Review by veeqtor — APPROVED** [review]

**Review by OKiMaureen — APPROVED** [review]

**Pull request closed without merging**